sycl/include/sycl/builtins.hpp

```diff
@@ -60,6 +60,10 @@ namespace __sycl_std = __host_std;
   __SYCL_BUILTIN_DEF(TYPE##3)                                                  \
   __SYCL_BUILTIN_DEF(TYPE##4)
 
+#define __SYCL_DEF_BUILTIN_GEOCROSSVEC(TYPE)                                   \
```

Comment:
Please add `#undef` for all new macros which was introduced in this patch

Reply:
Oh yes, forgot about it, thanks! [c72b5b2](https://github.com/intel/llvm/pull/10014/commits/c72b5b29146242886d34375e6f060b57443ad23e)